astro/src/content/quickstarts/quickstart-python-django-web.mdx

```diff
@@ -0,0 +1,258 @@
+
+`urls.py`
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-python-django-web/main/complete-application/mysite/app/urls.py"
+ lang="python" />
+
+### App Views
+
+Now overwrite the `app/views.py` file. There are three pages: the home page, the account page, and the make change page.
+
+`views.py`
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-python-django-web/main/complete-application/mysite/app/views.py"
+ lang="python" />
+
+Here's what these files do:
+
+* The `app` view, or home page, redirects the user to the `account` view if they are already logged in.
+* The `account` view checks whether the user is logged in. If the user is logged in, it returns the account page template, passing in the user's email address returned from FusionAuth. If the user isn't logged in, they are redirected to the login page. This also happens in the change page.
+* The `logout` view is necessary because you need to clear the session for the user in Django after the user logs out of FusionAuth. FusionAuth sends the user to this view after logging them out, which then returns them to the home page.
+* The `change` view looks longer only because it demonstrates how to do some calculations if the request is a POST instead of a GET. These calculations are then passed to the template in the `change` variable to display onscreen.
+
+## Customization
+
+The final step is to create the HTML pages and style them with CSS.
+
+* Copy the `static` folder with its contents from `complete-application/mysite/app/static` to your `/mysite/app/static` location.
+* Copy the `templates` folder with its contents from `complete-application/mysite/app/templates` to your `/mysite/app/templates` location.
+
+There are only two lines in these templates related to authentication. In the home template, there is a login button.
+
+```html
+<a class="button-lg" href="{% url 'oidc_authentication_init' %}">Login</a>
+```
+This URL is made available by the Mozilla plugin in the settings file. It directs the user to FusionAuth to log in, which then redirects back to your app account page.
+
+In the account and change pages, there is a logout form.
+
+```html
+<form id="logoutForm" class="button-lg"  action="{% url 'oidc_logout' %}" method="post">
+  {% csrf_token %}
+  <a class="button-lg" href="#" onclick="document.getElementById('logoutForm').submit();">Log out</a>
+</form>
+```
+Log out has to be a form that submits a POST request to FusionAuth. Here we use an anchor element that is easy to style, but if your users might disable JavaScript, you could use a traditional form `input` element.
+
+## Run the Application
+
+Update the default SQLite database before testing.
+
+```bash
+python manage.py migrate
+```
+
+You can now browse to [the Django app](http://localhost:8000/app).
+
+If the app is not running, you can use the following command.
+
+```bash
+source venv/bin/activate && python mysite/manage.py runserver
```

Comment:
Would I do this in a new window? Why do I need the `venv` command? Where should I run this from?

Didn't I start the server up above and you haven't told me to stop it? I'm confused.